2023/stylesheets/app.css

```diff
@@ -82,7 +82,7 @@ button, .button {
       color: #252C37;
       background-image: url("../assets/images/hex-button-border-disabled@2x.png"); }
   button.round, .button.round {
-    min-width: 180px;
+    min-width: 250px;
```

Comment:
This was for?

Reply:
This is to preserve styling @Saransh-cpp